tests/errmsgs/tyield_view_tuple.nim

```diff
@@ -0,0 +1,22 @@
+discard """
+  nimoutFormat: sexp
+  cmd: "nim check --msgFormat=sexp --filenames=canonical --hints:off $options $file"
+  action: reject
+"""
+
+proc make(x: var int): (int, var int) =
+  (0, x)
```

Comment:
"this code is ok because it returns a tuple constructed from the literal"

Reply:
The `make` procedure is only a helper and not related to the test. I added a doc comment for clarification.